pkg/hints/ec_hint.go

```diff
@@ -181,3 +182,78 @@ func computeSlope(vm *VirtualMachine, execScopes ExecutionScopes, idsData IdsMan
 
 	return nil
 }
+
+/*
+Implements hint:
+%{ from starkware.cairo.common.cairo_secp.secp256r1_utils import SECP256R1_ALPHA as ALPHA %}
+*/
+
+func importSecp256r1Alpha(execScopes ExecutionScopes) error {
+	execScopes.AssignOrUpdateVariable("ALPHA", SECP256R1_ALPHA())
+	return nil
+}
+
+/*
+Implements hint:
+%{ from starkware.cairo.common.cairo_secp.secp256r1_utils import SECP256R1_N as N %}
+*/
+func importSECP256R1N(execScopes ExecutionScopes) error {
+	execScopes.AssignOrUpdateVariable("N", SECP256R1_N())
+	return nil
+}
+
+/*
+Implements hint:
+%{
+from starkware.cairo.common.cairo_secp.secp256r1_utils import SECP256R1_P as SECP_P
+%}
+*/
+
+func importSECP256R1P(execScopes ExecutionScopes) error {
+	execScopes.AssignOrUpdateVariable("SECP_P", SECP256R1_P())
+	return nil
+}
+
+/*
+Implements hint:
+
+	%{
+	    from starkware.cairo.common.cairo_secp.secp_utils import pack
+	    from starkware.python.math_utils import ec_double_slope
+	    # Compute the slope.
+	    x = pack(ids.point.x, PRIME)
+	    y = pack(ids.point.y, PRIME)
+	    value = slope = ec_double_slope(point=(x, y), alpha=ALPHA, p=SECP_P)
+
+%}
+*/
+func computeDoublingSlopeExternalConsts(vm VirtualMachine, execScopes ExecutionScopes, ids_data IdsManager) error {
+	// ids.point
+	point, err := EcPointFromVarName("point", &vm, ids_data)
+	if err != nil {
+		return err
+	}
+
+	secp_p_uncast, err := execScopes.Get("SECP_P")
+	if err != nil {
+		return err
+	}
+	secp_p := secp_p_uncast.(big.Int)
```

Comment:
```suggestion
	secp_p, ok  := secp_p_uncast.(big.Int)
```
This cast can fail, please handle it